### ISSUE TITLE
not optimal if conditions: a false-positive

### ISSUE THREAD

**Issue by kalessil** [kalessil/phpinspectionsea#12]
Original link: https://bitbucket.org/kalessil/phpinspectionsea/issues/239
Author: @mfurlend

****The inspection is telling me that in this line:
```php
if ($_SERVER [ 'REQUEST_METHOD' ] === 'POST' && array_key_exists('comment',$_POST)) { ...
```
```php
array_key_exists('comment',$_POST)
```
is less expensive than 
```php
$_SERVER [ 'REQUEST_METHOD' ] === 'POST' 
```

Which is definitely not true (I benchmarked it to double check).
